exercises/grains/example.js

```diff
@@ -1,29 +1,33 @@
-var BigInt = require('./big-integer');
+var bigInt = require('./big-integer');
```

Comment:
Hmm. I really like `BitIng`, starting with uppercase.

I undersand your change, ESLint is complaining about that. Have you tried to use `new BigInt(2)` instead of `BigInt(2)`? It might be the same for this exercise and for the use of `BigInt` in this exercise. But, I think `BigInt` is a constructor function, so I'd prefer using `new` instead of starting it in lowercase.